start_admin_sshd.sh

```diff
@@ -65,31 +67,31 @@ if trusted_user_ca_keys=$(get_user_data_keys "trusted_user_ca_keys"); then
   ((++available_auth_methods))
 fi
 
-chown "${local_user}" -R "${user_ssh_dir}"
+chown "${LOCAL_USER}" -R "${USER_SSH_DIR}"
 
 # If there were no successful auth methods, then users cannot authenticate
 if [[ "${available_auth_methods}" -eq 0 ]]; then
-  user_data_condensed=$(tr -d '[:space:]' < "${user_data}")
+  user_data_condensed=$(tr -d '[:space:]' < "${USER_DATA}")
```

Comment:
This command would delete *intentional* whitespace as well as the (potential) extraneous space from read JSON. If the goal is to compact the JSON itself (and not user values), then let's use `jq` to do so - we already know its there and it has a handy flag for this:

```bash
# print out condensed json blob to logs.
jq --compact-output . "$USER_DATA"

# or handle fallback, when not json (I didn't read back to see if we 
# return early away from this path if its not known to be json):
jq --exit-status --compact-output . "$USER_DATA" 2>/dev/null || cat "$USER_DATA"
```

Reply:
That's fair. I'll definitely make this change, but I'd like to do so in a different PR to keep this limited to non-functional changes.

Reply:
Addressed in #25